<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_Main</class>
 <widget class="QMainWindow" name="frm_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logikgatter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Ressourcen.qrc">
    <normaloff>:/images/cyb_strudel_48x48.png</normaloff>:/images/cyb_strudel_48x48.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_operator_oben">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>25</y>
      <width>110</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: lightgrey;
border: 5px solid grey;</string>
    </property>
    <property name="text">
     <string>&amp;&amp; - AND</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="Radio_A1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1 - True</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="Radio_A0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0 - False</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="Radio_B1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1 - True</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="Radio_B0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0 - False</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="Radio_C2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0 - False</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="Radio_C1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1 - True</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_operator_unten">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>110</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: lightgrey;
border: 5px solid grey;</string>
    </property>
    <property name="text">
     <string>&amp;&amp; - AND</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_RandomA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_RandomB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RandomB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>255</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_RandomB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_Check_Quiz">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: black;
	background: lightgreen;
	border: 2px solid green;
}
QPushButton:hover {
	color: white;
	background: green;
	border: 2px solid green;
}</string>
    </property>
    <property name="text">
     <string>Auswerten</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ausgabe_oben">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>80</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;label_ausgabe_oben&lt;/p&gt;&lt;p&gt;label_ausgabe_oben&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_ausgabe_unten">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>255</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;label_ausgabe_unten&lt;/p&gt;&lt;p&gt;label_ausgabe_unten&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_newRandom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: black;
	background: lightgreen;
	border: 2px solid green;
}
QPushButton:hover {
	color: white;
	background: green;
	border: 2px solid green;
}</string>
    </property>
    <property name="text">
     <string>Neue Werte</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>110</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid grey;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>110</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid grey;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>110</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid grey;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>110</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid grey;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>110</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid grey;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>75</y>
      <width>110</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px solid grey;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_oben_punktC">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>69</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: lightgrey;
border: 3px solid grey;
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_unten_punktC">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>214</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: lightgrey;
border: 3px solid grey;
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="menu_File_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="menu_Help_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <property name="styleSheet">
    <string notr="true">/*QWidget {
	background : red;
}*/
QToolButton {
	margin:  2px;
	width: 45px;
	color: black;
	background : lightgrey;
	border : 2px solid grey;
}
QToolButton:hover {
	color: white;
	background : grey;
}</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menu_File_Quit">
   <property name="icon">
    <iconset resource="Ressourcen.qrc">
     <normaloff>:/images/quit.png</normaloff>:/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="menu_Help_About">
   <property name="icon">
    <iconset resource="Ressourcen.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Ressourcen.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
 </buttongroups>
</ui>
